dockerfile/README.md

```diff
@@ -11,7 +11,7 @@ The Dockerfile defines the resources and instructions to build the Docker image
  * Create a directory named `files` inside `DOCKERFILE_HOME`
      - This will result in a directory structure `<DOCKERFILE_HOME>/files`.
  
- * Add the JDK and WSO2 API Manager distributions
+ * Add the JDK and WSO2 Data Analytics Server distributions to `files` directory
```

Comment:
Shall we further update this document with documentation on 
"Modifying configurations of the Server running inside the container" ? An example documentation 
can be found @ https://github.com/wso2/docker-is/blob/master/dockerfile/README.md